### PR TITLE
Updated documentation for adding a new section

### PR DESCRIPTION
**Pull request by pjackson28**

### CODE REVIEW

**Comment by pjackson28** [comment]
@nschonni Any idea why this keeps failing MD009? I checked line 15 (and all other lines) and there are no trailing spaces.

File: docs/en/how-add-new-common-section-guidelines.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ The following are the steps for adding a new common section (e.g., checklist, im
 <!-- markdownlint-disable MD032 -->
 - **_data/playbook_schema.json and _data/guide_schema.json:** Add a property for the new section following the pattern of "checklist" and "guides"
 - **_config.yml:** Add an entry for the new section in the common strings section (following the pattern for "checklist" and "guides"). Also add the section key to guidelineSectionsOrder.
-- **_includes/filter-interface-retroaction.html:** Add a filter to the section filters for the new new section (following the pattern for "checklist" and "guides"
+- **_includes/filter-interface-retroaction.html:** Add a filter to the section filters for the new section (following the pattern for "checklist" and "guides"
+- **_includes/output-sections.html:** (Optional) Assign a glyphicon for the new section follow the pattern of "checklist" and "guides".
```
Comment (nschonni) [inline docs/en/how-add-new-common-section-guidelines.md]: There is a trailing space here

Comment (nschonni) [reply docs/en/how-add-new-common-section-guidelines.md]: Highlight and drag the line with your mouse and you'll see there is some sort of whitespace character at the end

Comment (pjackson28) [reply docs/en/how-add-new-common-section-guidelines.md]: Weird. I tried to do that earlier and got nothing at the end of the line. I even did a find to try to highlight any extra space and nothing was coming back for that line. I did find it this time so thanks for looking and sorry for not catching it myself.

**Pull request merged**